static/js/lib/learning_resources.js

```diff
@@ -228,3 +231,44 @@ export const filterItems = (
 
 export const privacyIcon = (privacyLevel: string) =>
   privacyLevel === "public" ? "lock_open" : "lock"
+
+export const isCoursewareResource = R.contains(R.__, [
+  LR_TYPE_COURSE,
+  LR_TYPE_PROGRAM,
+  LR_TYPE_BOOTCAMP
+])
+
+export const formatDurationClockTime = (value: string) => {
```

Comment:
seems like something moment should support itself!

Reply:
yeah, there's a closed issue on their repo, which people keep commenting on that it's not really what they needed: https://github.com/moment/moment/issues/463